.pre-commit-config.yaml

```diff
@@ -24,10 +24,23 @@ repos:
         types: [python]
         require_serial: true
 
+      - id: export-config-schema
+        name: export config schema
+        entry: blueapi config-schema -u
+        language: python
+        files: "src\/blueapi\/config.py|src\/blueapi\/utils\/"
+        pass_filenames: false
```

Comment:
```suggestion
```

I think we can replace this with a test like `test_schema_updated` to ensure that config_schema.json is always kept update